packages/amf/amf.0.1.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "Parser/serializer for Adobe's Action Message Format"
+maintainer: "Brian Caine <brian.d.caine@gmail.com>"
+authors: "Brian Caine <brian.d.caine@gmail.com>"
+license: "LGPL with OCaml linking exception"
+homepage: "https://briancaine.github.io/ocaml-amf/"
+bug-reports: "https://github.com/briancaine/ocaml-amf/issues"
+depends: [
+  "ocaml" {>= "4.01"}
+  "dune" {build & >= "1.0"}
+  "ppx_deriving" {build}
+  "ppx_sexp_conv" {build}
```

Comment:
I'm not entirely sure but I believe `ppx_sexp_conv` should not be a build dependency (it has a runtime dependency `ppx_sexp_conv.runtime`). I'm not entirely sure for `ppx_deriving` either. Can someone check this?

Reply:
cc @mseri if you know

Reply:
Your suspicions are correct, they both come with runtime. In general we decided some times ago that to avoid using the `{build}` for ppxs to avoid issues